yaml/nadav/yamel_exercise1_nadav.yaml

```diff
@@ -0,0 +1,65 @@
+# string
+name: Nadav
+# number
+age: 21
+isPerson: true
+# array
+food: [pizza, beer, schawarma]
+# array option 2
+friends:
+   - name: Aviv
+     age: 21
+     isPerson: true
+     isOlder: true
+
+    - name: Noy
+      age: 21
+      isPerson: true
```

Comment:
not true she is a psycho

Reply:
u stupid 
it's supposed to be "Itay", with an 'I'.